rasa_nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -29,6 +28,11 @@
 if typing.TYPE_CHECKING:
     import tensorflow as tf
 
+try:
+    import tensorflow as tf
+except ImportError:
+    logger.debug("No module named tensorflow")
```

Comment:
let's write a more helpful message here, maybe:

"Unable to import tensorflow. If you are not using the tensorflow pipeline, you can safely ignore this message. If you are using pipeline=tensorflow_embedding, this will create an error."